mobile/lib/common/settings/collab_close_screen.dart

```diff
@@ -125,7 +125,7 @@ class _CollabCloseScreenState extends State<CollabCloseScreen> {
 }
 
 RichText getCloseChannelText(ChannelStatusNotifier channelStatusNotifier) {
-  if (channelStatusNotifier.hasDlcChannel()) {
+  if (channelStatusNotifier.hasOpenPosition()) {
     return RichText(
         text: const TextSpan(
       style: TextStyle(fontSize: 18, color: Colors.black, letterSpacing: 0.4),
```

Comment:
The text below is weird (you didn't change it here but still). It should be:

```
Please, close your open position before collaboratively closing your channel
```

The comma is in the wrong place. Or just remove the comma altogether.